Accessibility Handbook.xcodeproj/project.pbxproj

```diff
@@ -1489,7 +1492,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Accessibility Handbook/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 278BFDM8KA;
```

Comment:
Hi @edequartel! Could you just remove your development team from the PR?

Reply:
I have changed it and added the file to git ignore.